buildSrc/src/main/kotlin/com.hedera.hashgraph.jpms-modules.gradle.kts

```diff
@@ -20,8 +20,6 @@ plugins {
 }
 
 extraJavaModuleInfo {
-    failOnMissingModuleInfo.set(true)
```

Comment:
`true` is the default here

Reply:
This was originally set to false and at some point was flipped back to true.